packages/suite/src/components/wallet/Fees/components/CustomFee.tsx

```diff
@@ -161,10 +157,7 @@ export const CustomFee = ({
         if (feeBig.isNaN()) {
             return 'CUSTOM_FEE_IS_NOT_NUMBER';
         }
-        // allow decimals in ETH since GWEI is not a satoshi
-        if (networkType !== 'ethereum' && !isInteger(value)) {
-            return 'CUSTOM_FEE_IS_NOT_INTEGER';
-        }
+
```

Comment:
Does it mean that decimal fee is now allowed for all bitcoin-like coins, cardano and ripple? Is it correct?

Reply:
Only for BTC (and I assume ETH, since it was there), done